src/Castle.Windsor.Extensions.MsDependencyInjection.Tests/WindsorScopedServiceProviderTests.cs

```diff
@@ -0,0 +1,17 @@
+namespace Castle.Windsor.Extensions.MsDependencyInjection.Tests
+{
+    using System;
+    
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.DependencyInjection.Specification;
+    
+    public class WindsorScopedServiceProviderTests : DependencyInjectionSpecificationTests
```

Comment:
This obviously runs the .NET DependencyInjection provided unit tests, so I think the project should probably be named `Castle.Windsor.Extensions.DependencyInjection.Specification.Tests`. This project is also not being compiled or run in the build.

And have a separate `Castle.Windsor.Extensions.DependencyInjection.Tests` project using NUnit for extra tests. e.g. support for Options and Logging libraries, and anything else specific to Windsor and for any defects that get found like has already been reported in the pull request but not picked up by this set.

Reply:
I can rename.

As for other unit tests - will add though i don't see reason having tests for Options and Logging - those are already tests by specification tests.

>This project is also not being compiled or run in the build.
It does for me. Are there any specifc steps for Castle? Becase the build setup for the solution is "unusual".

Reply:
> As for other unit tests - will add though i don't see reason having tests for Options and Logging - those are already tests by specification tests.

Great. Even if there are no extra unit tests we will at least have somewhere people can add unit tests before fixing any bugs.

> It does for me. Are there any specifc steps for Castle? Becase the build setup for the solution is "unusual".

Yep, it is definitely unusual, the build hacked around in the DNX days to make it work for the buggy tooling of the day, it probably needs to be refreshed to match what you'd do today if you were starting a clean project.

It looks like it actually is being built now, but the unit tests aren't being run. We are running the unit tests from `buildscripts/build.cmd`:

https://github.com/castleproject/Windsor/blob/b12304f975ffa832d1df12e4592dbe3e75058380/buildscripts/build.cmd#L55-L64